techmeet/data/templatetags/webpack_check.py

```diff
@@ -0,0 +1,14 @@
+"""Webpack Template Tags."""
```

Comment:
@thornycrackers you can remove this file if you want, I don't need it in the template for now.